src/bessel.jl

```diff
@@ -402,7 +402,7 @@ Scaled Bessel function of the first kind of order `nu`, ``J_\\nu(x) e^{- | \\ope
 """
 function besseljx(nu::Real, x::AbstractFloat)
     if x < 0 && !isinteger(nu)
-        throw(DomainErrorNoArgs)
+        throw(DomainError(x, "`x` cannot be negative and `nu` is not an integer."))
```

Comment:
This reads kind of weird. I'd say "`x` cannot be negative and `nu` must be an integer."